packages/vite/src/node/plugins/clientInjections.ts

```diff
@@ -35,6 +35,7 @@ export function clientInjectionsPlugin(config: ResolvedConfig): Plugin {
       const timeout = hmrConfig?.timeout || 30000
       const overlay = hmrConfig?.overlay !== false
       const isHmrServerSpecified = !!hmrConfig?.server
+      const hmrConfigName = path.basename(config.configFile || 'vite.config.ts')
```

Comment:
```suggestion
      const hmrConfigName = path.basename(config.configFile || 'vite.config.js')
```

I think we should fallback to `.js` like before.